### ISSUE TITLE
twitch stream names 

### ISSUE THREAD

**Issue by rogeraint** [henryperson/twitchmultivod#37]
it wont allow me to put the usernames as a way to search I don't know whats up with that please help

**Comment by UpgradeCourier** [comment]
yeah have the same issue, would love it if this function could be returned.
